frontend/compositions/dashboardHeader/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import { useMediaQuery } from './helperFunctions'
+import LogoMobile from './assets/LogoMobile.svg'
+import NPDCLogo from './assets/NPDCLogo.svg'
+import HamburgerMenu from './assets/Vector.svg'
+import DesktopNav from './desktopNav'
+import styles from './header.module.css'
+
+
+export default function DashboardHeader() {
+  const {
+    wrapper,
+    leftHeader,
+    logoContainer,
+    titleContainer,
+    rightHeader,
+    nav,
+    button,
+  } = styles;
+
+  const belowBreakpoint = useMediaQuery(375);
+  const logo = belowBreakpoint ? <LogoMobile /> : <NPDCLogo />;
+  const title = belowBreakpoint ? 'N.P.D.C.' : 'National Police Data Coalition';
+  const navBar = belowBreakpoint ? <HamburgerMenu /> : <DesktopNav />
+
+  return (
+    <header className={wrapper}>
+      <div className={leftHeader}>
+        <div className={logoContainer}>
```

Comment:
We have a `Logo` component now you can use! at `shared-components/logo.tsx` (you'll just have to affect the sizing)